example-code/app/package.json

```diff
@@ -13,7 +13,7 @@
     "extract": "betools extract --fileExt=ts --extractFrom=./src --recursive --out=../../generated-docs/extract",
     "docs": "npm run extract && npm run extract-assets",
     "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
-    "test": "npm run copy:test-backend-assets && npm run copy:backend-assets && betools test --defineWindow --testDir=\"./lib/backend/test\"",
+    "test": "npm run copy:test-backend-assets && npm run copy:backend-assets && mocha --no-config",
```

Comment:
Does the `no-config` option tell mocha to not look for an actual file and force it to use the one in the `package.json`?

Reply:
yeah, basically